security/openconnect/src/opnsense/service/templates/OPNsense/Openconnect/opnsense-openconnect

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# $FreeBSD$
+#
+# PROVIDE: opnsense-openconnect
+# REQUIRE: SERVERS
+# KEYWORD: shutdown
+#
+
+. /etc/rc.subr
+
+name=openconnect
+
+stop_cmd=openconnect_stop
+start_cmd=openconnect_start
+status_cmd=openconnect_status
+rcvar=openconnect_enable
+
+load_rc_config opnsense-openconnect
+pidfile=/var/run/${name}.pid
+command=/usr/local/sbin/${name}
+
+[ -z "$openconnect_enable" ] && openconnect_enable="NO"
+
+# status of openconnect
+openconnect_status()
+{
+	 if [ -n "$rc_pid" ]; then
+	     echo "${name} is running as pid $rc_pid."
+	     return 0
+	 else
+	     echo "${name} is not running."
+	 fi
+}
+
+# stop openconnect
+openconnect_stop()
+{
+	echo "stopping openconnect"
+	killall openconnect
+	ifconfig ocvpn0 destroy
+	return 0
+}
+
+# start openconnect
+openconnect_start()
+{
+{% if helpers.exists('OPNsense.openconnect.general.enabled') and OPNsense.openconnect.general.enabled == '1' %}
```

Comment:
we can't template this script, templating the /etc/rc.conf.d/opnsense-openconnect file should be enough to transport the settings for this

Password could be dumped as a separate file in 500 mode and then cat $file | pushed..

Reply:
https://secure.php.net/manual/de/function.proc-open.php

Reply:
this is still true, we cannot template this script, it needs to go to /usr/local/etc/rc.d/ directly